server/datastore/cached_mysql/cached_mysql.go

```diff
@@ -51,12 +51,12 @@ func (ds *cachedMysql) getFromRedis(key string, v interface{}) error {
 
 	data, err := redigo.Bytes(conn.Do("GET", key))
 	if err != nil {
-		return errors.Wrap(err, "getting value from cache")
+		return fmt.Errorf("getting value from cache: %w", err)
 	}
 
 	err = json.Unmarshal(data, v)
 	if err != nil {
-		return errors.Wrap(err, "unmarshaling object from cache")
+		return fmt.Errorf("unmarshaling object from cache: %w", err)
```

Comment:
Why not do `ctxerr.Wrap` on these redis and json errors?

Reply:
There is no context here (and the error is ignored at the call-site, so I guess even the wrapping is unnecessary, but yeah I didn't want to change the code more than I have to).